### ISSUE TITLE
Customize aws profile in provider.

### ISSUE THREAD

**Issue by ShylajaDevadiga** [rancher/rke2#28]
Few issues found during the initial install of rke2
1. aws profile is hardcoded, had to manually adjust the tf files to set the correct profile 
```
provider "aws" {
  region  = "us-east-2"
  profile = "rancher-eng"
}
```

2. With terraform version 0.12.26 warning is seen with interpolation syntax. terraform version 0.11 or earlier accepts below syntax.
```
Warning: Interpolation-only expressions are deprecated

  on main.tf line 238, in resource "aws_lb_target_group_attachment" "rke2-nlb-attachement":
 238:   target_group_arn = "${aws_lb_target_group.rke2-master-nlb-tg.arn}"
```
3. Build process fails while generating kubeconfig, could be due to reuse of ip for ec2 instance
```
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
null_resource.get-kubeconfig (local-exec): Host key verification failed.
```

**Comment by davidnuzik** [comment]
@ShylajaDevadiga can you please re-test - this is still an issue? NOT a priority - other priority/critical-urgent issues have precedence.

**Comment by ShylajaDevadiga** [comment]
aws profile needs to be set to specific organization like rancher-eng/rancher-qa depending on the user requirement.
Issue on interpolation is resolved resolved using terraform 0.11.
